tests/test_backend.py

```diff
@@ -164,6 +217,73 @@ def test_cache_value_error(self):
             b.cache = 8
 
 
+class TestBackendArchive(unittest.TestCase):
```

Comment:
The test case is ok but I would make it generic. Subclassing it for each backend should be easy.